content/src/fieldConfig/config.json

```diff
@@ -367,7 +367,27 @@
     "emptyCalendarBodyText": "If you are an LLC, C Corp, L,P, or LLP, you need to tell us your business formation date so we can tell you when your next filing is due. You can do this by [updating your profile](#).",
     "calendarGridViewButton": "Grid View",
     "hideOpportunityText": "Hide",
-    "calendarSnackbarHeading": "Calendar Added!"
+    "calendarSnackbarHeading": "Calendar Added!",
+    "taxCalendarAccessBody": "Get access to your tax calendar",
+    "taxCalendarAccessButton": "Get Access",
+    "taxCalendarModalPreviousButton": "Cancel",
+    "taxCalendarModalNextButton": "Save",
+    "taxCalendarModalErrorHeader": "Review the fields below for errors or omissions",
+    "taxCalendarModalHeader": "See Your Tax Due Dates",
+    "taxCalendarModalBody": "To access your tax records, we need to verify your information and get you access to New Jersey Department of Treasury's `Gov2Go|gov2go` platform.",
+    "taxCalendarModalBusinessFieldHeader": "#### Business Name",
+    "taxCalendarModalBusinessFieldErrorName": "Business Name",
+    "taxCalendarModalBusinessFieldMarkdown": "Your business name must match exactly with the name listed on your `Certificate of Formation or Authority|certificate-formation`.",
+    "taxCalendarModalTaxErrorName": "New Jersey Tax ID",
+    "taxCalendarModalTaxIdHeader": "`#### New Jersey Tax ID|tax-id`",
+    "taxCalendarModalTaxIdMarkdown": "Enter the 12-digit number below (xxx-xxx-xxx/xxx). \n\n (You can find your Tax ID listed as your 'Filer ID' in your email confirmation subject 'Confirmation of New Jersey Business Registration').",
+    "taxCalendarFailedErrorMessageMarkdown": "Review your Tax ID and/or business name for a match with the information in your Business Registration Certificate. If you need assistance, you can chat with one of our business advocates.",
+    "taxCalendarFailedBusinessFieldHelper": "Enter your correct business name",
+    "taxCalendarFailedTaxIdHelper": "Enter a correct 12-digit NJ Tax ID",
+    "taxCalendarFailedUnknownMarkdown": "There was a problem with your verification process, please try again later.",
+    "taxCalendarPendingCopyMarkdown": "#### Your Tax Calendar is pending. \n\n Your tax due dates are being added, check back in **7 days**.",
+    "taxCalendarSnackbarSuccessHeader": "Tax Calendar Added!",
+    "taxCalendarSnackbarSuccessBody": "Your businesses tax deadlines are now in your calendar."
```

Comment:
just from a content team perspective, might be nice if all `the taxCalendar` configs were their own key, which would make it easier to use in the CMS and also make all the keys shorter